petrel/petrel/cmdline.py

```diff
@@ -42,8 +42,9 @@ def submit(sourcejar, destjar, config, venv, name, definition, logdir, extrastor
     storm_class_path = [ subprocess.check_output(["storm","classpath"]).strip(), destjar ]
     if extrastormcp is not None:
         storm_class_path = [ extrastormcp ] + storm_class_path
-    storm_home = os.path.dirname(os.path.dirname(
-        subprocess.check_output(['which', 'storm'])))
+    storm_home = os.environ['STORM_HOME']
```

Comment:
OK. I'll change .gitignore. I have the source code for petrel checked out in /opt/petrel, but also installed it via `python config.py install`, hence it links to `/usr/local/bin`, but the script that build topologies seems to want the /opt/petrel (checked out code)? Might have misunderstood this.